src/ip_grid_descriptor_mod.f90

```diff
@@ -1,3 +1,13 @@
+!> @file
+!! @brief Uses derived type grid descriptor objects to abstract away the raw Grib-1 and Grib-2 grid definitions.
+!! @author Kyle Gerheiser
```

Comment:
The file-level author should be the programmer who created this code file. Was that you?

Reply:
Yes